WordPress/Classes/Services/AppUpdate/AppUpdateCoordinator.swift

```diff
@@ -24,7 +25,8 @@ final class AppUpdateCoordinator {
         remoteConfigStore: RemoteConfigStore = RemoteConfigStore(),
         isJetpack: Bool = AppConfiguration.isJetpack,
         isLoggedIn: Bool = AccountHelper.isLoggedIn,
-        isInAppUpdatesEnabled: Bool = RemoteFeatureFlag.inAppUpdates.enabled()
+        isInAppUpdatesEnabled: Bool = RemoteFeatureFlag.inAppUpdates.enabled(),
+        delayInDays: Int = 7
```

Comment:
Can we delay it by more days, just to be safe? I'd probably go with a month to let users a chance to update manually too. I don't think we need to start bombarding them with the updates right away, do we – give them a chance to update manually too.

Reply:
This value doesn't represent how many days to wait in between showing prompts.

It represents how many days to wait until a version has been released to show a prompt to update. I chose 7 days because phased release is complete after 7 days. This delay ensures that the prompt isn't shown during the phased release period - it prevents showing the prompt to users who have automatic updates enabled.

As discussed in p1716217030223529-slack-C072JBZL84U, my plan is change the flexible update interval remote config value from 5 to 90 days.